guides/advanced_guides/relevancy.md

```diff
@@ -0,0 +1,204 @@
+
+#### Examples
+
+:::: tabs
+
+::: tab Typo
+
+![Image from alias](../../public/image/vogli3.png)
+
+### Typo
+
+- `vogli` : 0 typo
+- `volli` : 1 typo
+
+The typo sorts the results by ascending typos on matched query words.
+
+Because `typo` is set as the first rule of our ranking rules, the number of `typos` sorts the document before
+
+:::
+
+::: tab Words
+![Image from alias](../../public/image/new_house.png)
+
+### Words
+
+Matched words :
+- **The Housemaid**: `Housemaid`, `new`, `house`, `housemaid`, `housework`: 5 words.
+- **The open House**: `House`, `new`, `house`: 3 words.
+
+The `words` ranking rule sorts the results by descending number of matching query words.
+
+:::
+
+::: tab Proximity
+![Image from alias](../../public/image/new_road.png)
+
+### Proximity
+
+The reason why `Creature` is listed before `Mississippi Grind` is because of the `proximity` rule.
+The smallest **distance** between the matching words in `creature` is smaller than the smallest **distance** between the matching words in `Mississippi Grind`
+
+The proximity rule sorts by descending order of distance length between two matches.
+:::
+
+::: tab Attribute
+![Image from alias](../../public/image/belgium.png)
+
+### Attribute
+
+`It's Tuesday, This must be Belgium` is first because the matched word: Belgium, is found in the `title` attribute and not the `description`.
+
+The `attribute` rule sorts by ascending [attribute importance](/guides/advanced_guides/relevancy.md#attributes-importance).
+
+:::
+
+::: tab Words position
+
+![Image from alias](../../public/image/belgium.png)
+
+### Word position
+
+`Gangsta` appears sooner than `Dunkirk` because `Belgium` appears sooner in the attribute.
+
+The `word position` rule orders by ascending matching word's index number.
+
+:::
+
+::: tab Exactness
+![Image from alias](../../public/image/knight.png)
+
+**Exactness**
+
+`Knight Moves` is displayed before `Knights of Badassdom` because `Knight` is a finished word, exactly the same as the search query. While with `Knights`, the search query is just a prefix.
+
+:::
+
+::::
+
+## Order of the rules
+
+Depending on your needs, you might want to change the order in which the rules are processed.
+
+In this case, using the [settings route](/references/ranking_rules.md#add-or-replace-ranking-rules) of your index, you can change the ranking order of the sorting rules.
+
+```json
+[
+  "_typo",
+  "_attribute",
+  "_proximity",
+  "_words",
+  "_words_position",
+  "_exact"
+]
+```
+
+## Adding your rules
+
+New rules can be added to the existing list at any time and anywhere in the existing order.
+
+A custom rule lets you create a descending or ascending sorting rule on a given attribute.
+To create a rule, you will need to communicate the attribute on which the rule is created and the order in which it will be sorted: `Asc(attribute_name)` or `Desc(attribute_name)`.
```

Comment:
```suggestion
To create a rule, you need to communicate the attribute on which the rule is created and the order in which it will be sorted: `Asc(attribute_name)` or `Desc(attribute_name)`.
```